requirements.txt

```diff
@@ -9,7 +9,7 @@ statsmodels
 pyyaml
 pybids
 scikit-image
-transforms3d~=0.3.1
+transforms3d
```

Comment:
Blaming `requirements.txt`, it looks like `transforms3d` has been pinned from the start (https://github.com/spine-generic/spine-generic/pull/229). And, it looks like this pin was originally supposed to target the _minimum_ version (https://github.com/spine-generic/spine-generic/pull/229/commits/798763ca92be0a3c923fa6c6241abd5d73d75bbe).

I think this means that it's probably fine to remove this pin? `transforms3d` seems to update [relatively infrequently](https://github.com/matthew-brett/transforms3d/issues/46), too, so I imagine the auto-updates would mainly be when something breaks upstream and the author is releasing a fix.

Reply:
Thanks for digging! I think upgrading `transforms3d` looks safe enough. But it looks like it's never directly imported in `spine-generic`, only used (possibly?) indirectly through dependencies. In which case, without a version pin, maybe we don't have to mention `transforms3d` at all?